### ISSUE TITLE
Failed to validate the SSL certificate for osticket

### ISSUE THREAD

**Issue by plaindocs** [perryk/perryk-vagrant-osticket#2]
Not really sure how to fix this though. 

```
TASK [perryk.osticket.provision.dev : download osticket ldap module directly from osticket] ***
fatal: [default]: FAILED! => {"changed": false, "msg": "Failed to validate the SSL certificate for osticket.com:443. Make sure your managed systems have a valid CA certificate installed. You can use validate_certs=False if you do not need to confirm the servers identity but this is unsafe and not recommended. Paths checked for this platform: /etc/ssl/certs, /etc/pki/ca-trust/extracted/pem, /etc/pki/tls/certs, /usr/share/ca-certificates/cacert.org, /etc/ansible. The exception msg was: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:618)."}
```

**Comment by perryk** [comment]
Ahh ok, I've just now confirmed that a couple of pre-built osticket modules are no longer available for direct download from the osticket website. It was always a bit of just a lucky find that I stumbled across them there and I assume the devs always considered this files as an extra and not related to their official releases. Perhaps they changed websites and removed all of this.

I'll comment out these 2 tasks which download these 2 .phar files for the moment.

I'm not sure about going forward, the storage one is easy to obtain from here: https://github.com/osTicket/osTicket-plugins

This playbook would have to build and install it also, but from memory this was straightforward.

The same thing can be done for the LDAP module however I recall having a hard time building it. 

Potentially I could add these 2 .phar files to this repo, but I'd have to consider the implications of this first.

It looks like you will have to manually install those 2 modules if you want to test them out for now.

Thanks for letting me know about this. It has been a while since I looked at this setup and it is one of the first Ansible projects I did. I'd likely do things a bit different in terms of using more variables etc if I was to do this again.

Cheers.
